src/sentry/event_manager.py

```diff
@@ -593,6 +599,18 @@ def save(
                     project=project, event=job["event"], sender=Project
                 )
 
+            if (
+                has_event_minified_stack_trace(job["event"])
+                and not project.flags.has_minified_stack_trace
+                # We only want to record events from projects created after 2022-12-14,
+                # otherwise amplitude would receive a large amount of data in a short period of time
+                and project.date_added
+                > settings.START_DATE_TRACKING_FIRST_EVENT_WITH_MINIFIED_STACK_TRACE_PER_PROJ
+            ):
```

Comment:
Since we talk here about 2022-12-14 does it make any sense to have the `START_DATE_TRACKING_FIRST_EVENT_WITH_MINIFIED_STACK_TRACE_PER_PROJ` in settings ?
As far as I can see `START_DATE...` is only used here.
I would define it in this file (at the top), I would add the comment about 2022-12-14 there and use it here without
mentioning the date again.